.github/workflows/ci.yml

```diff
@@ -84,20 +84,36 @@ jobs:
         run: |
           ./gradlew ktfmtCheck
 
+      - name : new python venv
+        run: |
+          cd functions
+          python3 -m venv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
+          cd ..
```

Comment:
There's no need to go in and out of the directory, you can call the right commands from the roots directory to do this